pkg/cmd/server/start/master_args.go

```diff
@@ -219,7 +230,13 @@ func (args MasterArgs) BuildSerializeableEtcdConfig() (*configapi.EtcdConfig, er
 		StorageDir:    args.EtcdDir,
 	}
 
+	if args.ListenArg.UseTLS() {
```

Comment:
> GetEtcdBindAddress derives from ListenArg...

So.... I need to make sure that etcdAddr matches what will be produced by twisting up the listenAddr for my master?  I really want to be free of this, but do we need to try to figure out which configurations we want to support from the command line and then come up with some sane arguments to describe them?